.github/workflows/ci.yml

```diff
@@ -17,15 +17,24 @@ jobs:
   checks:
     name: ESLint and Prettier checks
     runs-on: ubuntu-latest
+    outputs:
+      wallet-back: ${{ steps.checker.outputs.wallet-back }}
+      wallet-front: ${{ steps.checker.outputs.wallet-front }}
+      boutique-front: ${{ steps.checker.outputs.boutique-front }}
+      boutique-back: ${{ steps.checker.outputs.boutique-back }}
+
     steps:
       - uses: actions/checkout@v4
+      - id: checker
+        uses: ./.github/workflows/file_changes
       - uses: ./.github/workflows/setup
       - run: pnpm checks
 
   build-wallet-frontend:
     name: WALLET - Build frontend
     runs-on: ubuntu-latest
     needs: checks
+    if: ${{needs.checks.outputs.wallet-front || 'false' == 'true'}}
```

Comment:
can this expression be changed to `!!needs.checks.outputs.wallet-front` or `needs.checks.outputs.wallet-front == true`?